python-sdk/src/astro/databases/google/bigquery.py

```diff
@@ -134,9 +134,36 @@ def default_metadata(self) -> Metadata:
         return Metadata(
             schema=self.DEFAULT_SCHEMA,
             database=self.hook.project_id,
-            region=BIGQUERY_SCHEMA_LOCATION,
         )  # type: ignore
 
+    def populate_table_metadata(self, table: BaseTable) -> BaseTable:
+        if (
+            table.temp
+            and (self.table and not self.table.metadata.is_empty())
+            and (table.metadata and table.metadata.is_empty())
+        ):
```

Comment:
Could this all be a function?

Reply:
It is already a small function that calls another function, don't see the need to have another function here